packages/playground/blueprints/src/lib/blueprint.ts

```diff
@@ -80,4 +80,9 @@ export interface Blueprint {
 	 * executed.
 	 */
 	steps?: Array<StepDefinition | string | undefined | false | null>;
+
+	/**
+	 * Print messages to the console.
+	 */
+	helloWorld?: string;
```

Comment:
For completeness – this is a new Blueprint _property_. A _step_ would be a part of the `steps` array. Either is fine, depending on what you hoped to achieve here, but I'm mentioning since you specifically used the word `step`.

Reply:
I did it in this way just to see if I can do it.
I will clean it up in case we want to use this as a example in the future.